edx_sga/templates/staff_graded_assignment/show.html

```diff
@@ -116,7 +119,7 @@
 
   <section aria-hidden="true" class="modal staff-modal" id="{{ id }}-grade" style="height: 75%">
     <div class="inner-wrapper" style="color: black; overflow: auto;">
-      <header><h2>{% trans "Staff Graded Assignment" %}</h2></header>
+      <header><h2 class="display_name"></h2></header>
```

Comment:
Also I would like to keep the "Staff Graded Assignment" that is trans'd when you fix this so it appears as suggesed: `{{ display_name }} - {% trans "Staff Graded Assignment" %}`